src/components/AddNewStationForm/AddNewStationForm.jsx

```diff
@@ -1,9 +1,18 @@
 import { useForm } from 'react-hook-form';
+import { useContext } from 'react';
 import cx from 'classnames';
 
 import styles from './AddNewStationForm.module.scss';
+import { StationsContext } from '../../context/StationsContext';
+
+export const AddNewStationForm = () => {
+  const {
+    setModalIsOpen,
```

Comment:
Variables holding boolean values and setter functions should follow a bit different naming convention: `isValue`/`setIsValue`, or even better: `isValue`/`setValue`. This lets you know at first glance that the value is indeed supposed to be a boolean: by the `is`/`are` keyword.

Sometimes people use words like `should`, for example `shouldComponentRender` or `shouldBeVisible`.

The rule of thumb is to use the verb as the first part of the variable name to let the reader know what kind of value it contains.

You already have `isModalOpen`, so you're on a good path :)